src/reducers/closeModal.ts

```diff
@@ -17,11 +15,10 @@ const modalRemindMeLater = (state: State, { id, duration = 0 }: { id?: string; d
     modals: {
       ...state.modals,
       [id ?? state.showModal!]: {
-        ...state.modals[id ?? state.showModal!],
-        hideuntil: time
+        ...state.modals[id ?? state.showModal!]
       }
```

Comment:
This is a NOOP

Reply:
This was existing code but seems we do not need this. Hence deleted.